exercises/grep/description.md

```diff
@@ -33,6 +33,17 @@ hello
 hello again
 ```
 
+If given multiple files, `grep` should prefix each found line with the file it was found in.
+As an example:
+
+```text
+input.txt:hello
+input.txt:hello again
+greeting.txt:hello world
```

Comment:
unlike with `input.txt` (whose contents are shown above), I do not believe it is necessary to explicitly show the contents of `greeting.txt`; the example is here just to show the format of the prefix (importantly that it contains a colon).

Reply:
Is it necessary to give an example when `-n` is also specified?

Reply:
That is a good idea, because we also have a test that does that. At the very minimum, a textual description that says the line number prefix goes after the filename prefix will be needed, so I'll add that. As for whether an example of that is needed, I could be convinced either way.